<launch>

	<node name="utm_odometry_node" pkg="nautonomous_pose_odom" type="utm_odometry_node" output="screen">
		
		<!-- Inputs -->
  		<remap from="fix" to="/pose/utm/fix" />
		
		<!-- Parameters -->
  		<param name="rot_covariance" value="99999" />
  		<param name="frame_id" value="base_footprint" />

		<!-- Outputs -->
		<remap from="odom" to="/pose/utm/odom"/>

	</node>

</launch>
